openbas-front/src/utils/Environment.js

```diff
@@ -8,7 +8,7 @@ import { useLocation } from 'react-router-dom';
 const MESSENGER$ = new Subject().pipe(debounce(() => timer(500)));
 export const MESSAGING$ = {
   messages: MESSENGER$,
-  notifyError: (text) => MESSENGER$.next([{ type: 'error', text }]),
+  notifyError: (text, sticky) => MESSENGER$.next([{ type: 'error', text, sticky }]),
```

Comment:
Should we have a default value for sticky ?
Or it's already handle in the class ?

Reply:
As of now, we're using the fact that sticky is undefined and therefore falsy. That would be better to have a default value tho, so I'm changing it.

Reply:
Done